src/adam_bashforth_adaptive.jl

```diff
@@ -0,0 +1,371 @@
+####################################
+# Explicit/Implicit Adaptive Adam-Bashforth solvers
+####################################
+# (Hairer & Wanner 1996, Vol I, p.398-400, 421-423)
```

Comment:
Yes, I should include the section on the fixed step AB method as background information.